src/qt/configurenamedialog.cpp

```diff
@@ -98,4 +111,13 @@ void ConfigureNameDialog::onDataEdited(const QString &name)
 {
     ui->dataSize->setText(tr("%1 / %2").arg(name.size()).arg(MAX_VALUE_LENGTH_UI));
     ui->dataSize->resize(ui->dataSize->fontMetrics().horizontalAdvance(ui->dataSize->text()), ui->dataSize->height());
+
+    std::string data = ui->dataEdit->text().toStdString();
+
+    if(IsValidJSON(data)){
+        ui->labelValidJSON->setText(tr("Valid JSON text."));
```

Comment:
I suspect that showing text in the label when the JSON is valid will be distracting. Would it make sense to set the label to the empty string in this case instead?

Reply:
Makes sense

Reply:
Actually hmm, maybe this actually is beneficial, in the sense that if we later add support for CBOR as a more compact alternative to JSON, it could actually be informative. I'm going to tentatively retract this request.

Reply:
Actually I'll leave it up to you. :) I don't feel super strongly here.